src/services/TestResultsService.ts

```diff
@@ -4,6 +4,7 @@ import moment from "moment";
 import { IInvokeConfig } from "../models";
 import { Configuration } from "../utils/Configuration";
 import { LambdaService } from "./LambdaService";
+import { TestResultSchema } from "@dvsa/cvs-type-definitions/types/v1/test-result";
```

Comment:
We now have the defined type, so can use it in the signature for expandTestResults instead of any